client/blocks/reader-recommended-sites/index.jsx

```diff
@@ -49,7 +49,7 @@ export class RecommendedSites extends React.PureComponent {
 						map(
 							sites,
 							( site, index ) => {
-								const siteId = site.siteId;
+								const siteId = site.siteId || site.blogId;
```

Comment:
i could do a map higher up to turn blogId into siteId.  

I think ideally we could pick one and then change them all at the data-layer.

🤷‍♂️